contrib/coredns/policy/policy.go

```diff
@@ -119,6 +119,52 @@ func (p *PolicyMiddleware) getEDNS0Attrs(r *dns.Msg) ([]*pb.Attribute, bool) {
 	return attrs, foundSourceIP
 }
 
+
+type LocalResponseWriter struct {
+	dns.ResponseWriter
```

Comment:
You don't need to include it here in order for the struct to implement the interface. Just having the right methods defined does it.